qlib/backtest/high_performance_ds.py

```diff
@@ -413,6 +426,11 @@ def sum_all_indicators(
         for metric in metrics:
             tmp_metric = PandasSingleMetric({})
             for indicator in indicators:
-                tmp_metric = tmp_metric.add(indicator.data[metric], fill_value)
+                if(metric == "trade_price"):
```

Comment:
The implementation doesn't align with the name `sum_all_indicators`.

Reply:
amount_mult_price

Reply:
Way1
- (trade_price * deal_amount).sum() / deal_amount.sum()

way 1
- new field:  trade_price * deal_amount
- sum:  new filed,  deal_amount 
- new filed / deal_amount 

make sum more clear